dataprofiler/profilers/base_column_profilers.py

```diff
@@ -247,6 +247,21 @@ def report(self, remove_disabled_flag: bool = False) -> dict:
         """
         raise NotImplementedError()
 
+    def parse(self, data):
```

Comment:
class method maybe so we don't have to init before?

```python
@classmethod
def load_from_json(cls, data):

    profile = cls(data["name"])
    for attr...:
        setattr(profile, attr, value)
```


```python
int_profile = IntProfile.load_from_json(json_dict)
```

Reply:
done